network/p2p/p2pnode/libp2pNode_test.go

```diff
@@ -422,6 +479,7 @@ func ensureSinglePairwiseConnection(t *testing.T, nodes []p2p.LibP2PNode) {
 			if this == other {
 				continue
 			}
+			fmt.Println(fmt.Sprintf("%s -> %s", this.Host().ID(), other.Host().ID()), this.Host().Network().ConnsToPeer(other.Host().ID()))
```

Comment:
It seems this line is a debug left over; please remove it.

Reply:
https://github.com/onflow/flow-go/pull/4022/commits/f1335e2e71ae8cd75cd87546f0cdbe729f247512